src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -976,6 +980,29 @@ private void declareNamespace(
     }
   }
 
+  /**
+   * Special emit for known aliases See KNOWN_ALIASES comment why this workaround is need.
+   *
+   * <p>This shares some similarity with visitTypeValueAlias method above, but it has to hardcode
+   * one assumptions - the alias is for a class with no generics.
+   */
+  private void visitKnownTypeValueAlias(String unqualifiedName, String alternativeAliasName) {
```

Comment:
Any reason to move this?

Reply:
Alas, yes.  I had to call it from declareNamespace(), which isn't part of TreeWalker.